overrides.json

```diff
@@ -17,5 +17,8 @@
     "chrome",
     "60",
     false
-  ]
+  ],
+  "https://github.com/mdn/browser-compat-data/pull/17365#issuecomment-1216459968",
+  ["api.FontFaceSetLoadEvent", "chrome", "35", true],
```

Comment:
Will this override only change what the results in Chrome 35 are, causing a flip-flopping inferred statement?

Reply:
If anything, this will create a complex inferred statement, and thus end up not updating the data.  Maybe we should allow for setting a range as the version (ex. `aa-bb`) so we don't have to duplicate this statement so many times?